### ISSUE TITLE
Fix date handling

### ISSUE THREAD

**Issue by RichardJECooke** [bukosabino/ta#107]
This step in your docs `ta.utils.dropna` gives errors on my date column like:
`TypeError: Invalid comparison between dtype=datetime64[ns] and float` if it's a datetime or 
`string and float` if it's a string represenation of a timestamp.

because these lines expect everything to be a number:
```
df = df[df < math.exp(709)]  # big number
df = df[df != 0.0]
```

They need to handle different datatypes correctly without crashing the app

**Comment by bukosabino** [comment]
Hi @RichardJECooke  ,

I have worked in a solution dropping values only in numeric columns.

Please, take a look to the updated code, and let me know if you have any additional problems: https://github.com/bukosabino/ta/blob/develop/ta/utils.py

Thank you for the advice!

Best,
Dario

**Comment by bukosabino** [comment]
Hi @RichardJECooke ,

This feature has been included in the last version of the library (v0.5.11).

To install the last ta version you can do:

```sh
pip install -U ta
```
